docs/guidelines/postman.md

```diff
@@ -0,0 +1,122 @@
+
+## Benefits of being logged in with an user account
+Logging in with a user account enables most of the features within Postman, including:
+
+- Workspaces
+- Collections
+- Sharing and collaboration
+- **Automatic synchronization to the cloud / backup**
+
+However, the automatic synchronization to the cloud imposes both security and regulationary concerns. Here are the results of a _a simple_ Threat Model exercise:
+
+| Threat | Description | Mitigation |
+|---|---|---|
+|Information disclosure|The API being tested has sensitive data in the responses - could be `confidential` or `personal` data - which would automatically be uploaded to the cloud|Know what data classification the data in the **requests and responses** belongs to and if uncertain, try to avoid using Postman. When using automatic testing, ensure tests are being performed against a "synthetic" test environment|
+|Account takeover / session hijacking|The account is breached and malicious actors can log in to Postman using your account|Only use SSO as there are protective and preventive measures in place to avoid/detect malicious actors logging in|
+|Data breach at Postman servers|A malicious actor has obtained data residing on the Postman servers|Avoid storing sensitive data that is being synchronized towards Postman servers|
+|Credentials stored unencrypted|Storing credentials in plain-text fields, in local plain-text files|Use proper solutions when handling credentials, like Postman Vault or use OAuth 2.0 authorization flows without client secrets (require user to log in through SSO)|
+
+
+## Best practices
+The automatic synchronization to the cloud imposes both security and regulationary concerns.
+Following the best practices will mitigate some of those concerns.
+
+### SSO
+_Never_ use a private or non-Equinor account in an Equinor context.
+If needing the full feature set of Postman, request access through AccessIT and use SSO for login. 
+This ensures 
+  - that the proper security checks on the account can be enforced (e.g. avoid logins from completely strange IPs, known malicious IP)
+  - that access control is properly enforced
+  - that access can properly be revoked
+  - that policies are followed and are enforced
+  - and others ...
+
+### Avoid storing sensitive data (in an insecure way)
+If needing to store credentials or other sensitive data, use [Postman Vault 🔗](https://learning.postman.com/docs/sending-requests/postman-vault/postman-vault-secrets/). Note that the latter is _not_ being synchronized with Postman and anything in the Vault remains local.
+
+If sensitive data is being stored in the environment, you should set the type to "secret". This will mask the input.
+
+!!! important "Be careful"
```

Comment:
This is very important. Should we add a line on this in the threat model overview? Should we suggest that "people" now and then export their stuff and examine and verify what's exposed is ok?

Reply:
Yes!
Added some reflections